esp-hal/src/dma/mod.rs

```diff
@@ -1690,7 +1690,7 @@ where
     }
 
     fn stop_transfer(&mut self) {
-        R::stop_out()
+        R::stop_out();
```

Comment:
This is unnecessary and now `stop_transfer` is inconsistent with the IRQ-related functions a bit below.

Reply:
I swear I don't see that in the diff

Reply:
Ohhhh, you're looking at the merge commit

Reply:
Yeah you're right, apologies.

Reply:
I didn't realize the difference between my stop PR and this one.

I opted to make it look like I didn't touch the DMA code in this PR. Should I remove the semicolon in this PR?

Reply:
No, let's collect minor eyesores and clean them up in one go in the future.